api/src/main/java/org/apache/cloudstack/api/command/user/firewall/CreatePortForwardingRuleCmd.java

```diff
@@ -358,7 +358,7 @@ public void create() {
             setEntityId(result.getId());
             setEntityUuid(result.getUuid());
         } catch (NetworkRuleConflictException ex) {
-            s_logger.info("Network rule conflict: " + ex.getMessage());
+            s_logger.info("Network rule conflict: ", ex);
```

Comment:
why are we logging the same thing twice here?

Reply:
Yeah, the same thing are logged twice, although they are different levels... I'm confused too, do you suggest to remove one from them? The trace one or the info one

Reply:
I would suggest removing the info one because we have a throw exception that we can send the `ex` exception.

BTW: thanks for your PRs lately. We really need this tasks of cleaning up and standardization of log messages an exceptions.

Reply:
Done it, thanks for your suggestion 👍